src/partials/subheading/do-rss.hbs

```diff
@@ -0,0 +1,9 @@
+<a class="release-notes-rss link-subheading align-center flex" href="https://docs.mulesoft.com/releases.rss" title="Add this URL to your RSS feed reader to stay up to date with MuleSoft releases.">
```

Comment:
just a question...`align-center` doesn't seem to be doing anything here - I removed it, and the element still looks the same. Does it do anything?

(I am ok leaving it as-is though)

Reply:
Good catch. This was a copy-paste from another hbs template and I forgot to remove it. 

Removed!